analysis/queries.py

```diff
@@ -30,6 +30,7 @@ def get_experiment_data(experiment_names):
         .select_from(Experiment)\
         .join(Trial)\
         .join(Snapshot)\
-        .filter(Experiment.name.in_(experiment_names))
+        .filter(Experiment.name.in_(experiment_names))\
+        .filter(~Trial.preempted.is_(True))
```

Comment:
Why not
Trial.preempted.is_(False)

Reply:
I could make this work, but I would have to change the migration to add preempted=False to each trial. Should I do this? Otherwise this query fails when combining new data with old because all of the old trials have null in this column.

Reply:
nah, not needed then.

Reply:
I'll try, I'd rather not have to contort our code forever just because the migration wasn't thorough enough.